balancer-js/src/lib/constants/config.ts

```diff
@@ -28,6 +29,9 @@ export const BALANCER_NETWORK_CONFIG: Record<Network, BalancerNetworkConfig> = {
         weightedPoolFactory: '0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9',
         composableStablePoolFactory:
           '0xdba127fBc23fb20F5929C546af220A991b5C6e01',
+        erc4626LinearPoolFactory: '0x67A25ca2350Ebf4a0C475cA74C257C94a373b828',
+        // aaveLinearPoolFactory: '0xf23b4DB826DbA14c0e857029dfF076b1c0264843', // This address is not right
```

Comment:
Think we can update this now?
Do we have addresses for the other Linear factories?

Reply:
No, this aaveLinearPoolFactory address still wrong in the docs :/ will remove it and put it back when it's fixed.
About other linear factories, I wrote down every linear pool factory that were in the balancer docs(and beets for Fantom Network), there is another place where I can find the factory addresses?

Reply:
Update: The aaveLinearPoolFactory address was fixed, and nico shared a link with the aaveLinearPoolFactory address in the monorepo, so now I know how to find other deployment addresses, will check it out and update it